setup.cfg

```diff
@@ -72,6 +72,8 @@ exclude =
     mitmproxy/proxy/config.py
     mitmproxy/proxy/server.py
     mitmproxy/proxy/layers/tls.py
+    mitmproxy/proxy/layers/dtls.py
```

Comment:
I am adding this file because the DTLS layers are simply sub-classes of TLS layers with some copy and pasted code. I'm unable to copy the `*TLSLayer` tests because they use the `ssl` module which does not support DTLS. It didn't seem worth the time to write tests (creating another version of `SSLTest` that works for DTLS) for the new layers since they are practically identical to the TLS layers which we are already testing. Open to suggestions of course.

Reply:
Given how (fantastically) minimal the changes in the DTLS layer now are I think it would make most sense to just merge the TLS and DTLS layers? I think you should be able to add something like
```python
@property
def is_dtls(self):
    return self.conn.transport_protocol == "udp"
```
and then get by with very few if-else constructs in the layers. This would also get around the test coverage issue. I will leave it up to you if you want to leave the `parse_client_hello` stuff in `layers/dtls.py` (we can add a top-level comment there pointing to `tls.py`) or rename it and integrate it into `tls.py` as well. Both is fine with me. :)